cli.js

```diff
@@ -28,6 +28,8 @@ try {
 					process.exit(1);
 				}
 
+				val = Math.round((val) * 10) / 10;
```

Comment:
Yeah, I'm going to do some cleanup later. Just wanted to get a fix out directly.